src/main/java/com/aws/greengrass/mqttclient/MqttClient.java

```diff
@@ -83,10 +85,15 @@
     static final String MQTT_OPERATION_TIMEOUT_KEY = "operationTimeoutMs";
     static final int DEFAULT_MQTT_OPERATION_TIMEOUT = (int) Duration.ofSeconds(30).toMillis();
     static final String MQTT_MAX_IN_FLIGHT_PUBLISHES_KEY = "maxInFlightPublishes";
-    static final int DEFAULT_MAX_IN_FLIGHT_PUBLISHES = 1;
+    static final int DEFAULT_MAX_IN_FLIGHT_PUBLISHES = 5;
     public static final int MAX_SUBSCRIPTIONS_PER_CONNECTION = 50;
     public static final String CLIENT_ID_KEY = "clientId";
     public static final int EVENTLOOP_SHUTDOWN_TIMEOUT_SECONDS = 2;
+    static final int DEFAULT_PUBLISH_RETRIED_COUNTER = 100;
+    // http://www.steves-internet-guide.com/mqtt-broker-message-restrictions/#:~:text=The%20MQTT%20protocol%20allows%20messages,especially%20public%20brokers%2C%20will%20restrict.
```

Comment:
Link to the actual specification. http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html#_Toc398718023